### ISSUE TITLE
SLAC output interpretation

### ISSUE THREAD

**Issue by AdenoV** [veg/hyphy#879]
Hi everyone,
I am doing some selection analysis on datamonkey server. In this case I have a doubt about SLAC output table. ¿Is there some scientific article or webpage were I can found an explanation about the output table?
On the other hand, I have obtained dn-ds values below 0. Also some very high values of dn and ds for some genes (320, 6333 ....) ¿Is it posible?
JR

**Comment by spond** [comment]
Dear @AdenoV,

Take a look at http://hyphy.org/tutorials/current-release-tutorial/ and http://hyphy.org/resources/hyphybook2007.pdf 

There is nothing remarkable about `dN - dS < 0` (note that this is **not** the dN/dS ratio). Negative `dN-dS` is indicative of negative selection.

`dN` and `dS` values are not computed for genes, but rather for individual sites. Very high values like you report would be unusual. 

Finally, I would suggest you use FUBAR in place of SLAC in general practice.

Best,
Sergei

**Comment by AdenoV** [comment]
Dear Sergei,
I appreciate your advicies. I though dn-ds in the SLAC output colum was dN/dS. I want to analyze selection pressures on different viral genes. I ran an SLAC analysis to see that and then I calculated the dN dS average and I also want to calculate dN/dS for each gene. Probably, as you said, it could be better to use FUBAR, becaude this estimations or average are not correct. 
On the other hand, I obtained the following dN and dS media values for each gene:

dS    | dN 
-- | --
12,6541131 | 6,67532923
9,73107398 | 5,39300608
18,3233552 | 7,31442079
6,97497572 | 3,02227802
15,0156424 | 4,28580584
18,8904632 | 5,16327581
12,8523108 | 3,27177135
14,8080724 | 3,51725063
16,3953304 | 4,20819772
16,3953304 | 4,20819772
12,9406132 | 3,22393579
9,63238601 | 3,34263033
9,40370969 | 2,98803574
9,25289635 | 2,0874694
27,7731976 | 5,83426311
14,6112663 | 4,9434412
16,6867934 | 5,47887766
16,6867934 | 5,47887766
131,277328 | 34,6579034
8,91262284 | 5,09223293
24,9529329 | 7,31943423
6,23755151 | 2,89616242
11,1776762 | 5,22856549
5,34112064 | 3,27625292
6073,28569 | 3430,52123
0,52063892 | 0,31367612
6,24533243 | 2,94696285
10,8322794 | 2,61041347
10,1753883 | 4,58604412
235,99326 | 93,9707147
3,74160744 | 2,36325756
14,7307039 | 5,46747682
8,36685505 | 2,96806213
5,87872789 | 1,94612285
11,8523481 | 4,46454204
15,0436204 | 3,94220186
124,171741 | 62,453489
6,84505315 | 3,12293226
1,16569304 | 1,04844455
9,85499926 | 4,70604072

However, I do not know why I obtained some high values of them.

Cheers,
JR
